src/experimental/dx-standalone-linux/test/verify-assets.sh

```
@@ -0,0 +1,69 @@
+
+echo "[verify] #2 — no inline comments in dx-standalone.env (systemd keeps them in the value)"
+if grep -qE '=[^#]*[[:space:]]+#' "$SYSD/dx-standalone.env"; then
+  no "dx-standalone.env has a value with a trailing inline comment"
+  grep -nE '=[^#]*[[:space:]]+#' "$SYSD/dx-standalone.env" | sed 's/^/       /'
+else
+  ok "dx-standalone.env: every comment on its own line"
+fi
+
+echo "[verify] unit ExecStart points at the launcher (native binary reality)"
+grep -q 'ExecStart=/usr/local/bin/standalone-pem' "$SYSD/standalone-pem.service" \
+  && ok "standalone-pem.service execs the launcher" \
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)